autoscale-dockerswarm-container/pom.xml

```diff
@@ -24,7 +24,7 @@
         <version>1.5.0-SNAPSHOT</version>
     </parent>
     <artifactId>autoscale-dockerswarm-container</artifactId>
-    <packaging>docker</packaging>
+    <packaging>pom</packaging>
```

Comment:
I'm fine with this but I suspect it means you have to add the executions, like in the other container project.

Reply:
Changed back to docker.

Reply:
To ignore maven dependency:analyze goal, it is changed from docker to pom.This was caused the jenkins build to fail.Now the build is passed.
http://sou-jenkins2.hpeswlab.net/job/Autoscaler/view/Developer/job/Autoscaler~autoscaler~SCMOD-5060~CI/49/console